test/units/end.sh

```diff
@@ -4,13 +4,13 @@
 set -eux
 set -o pipefail
 
-(! journalctl -q -o short-monotonic --grep "didn't pass validation" | grep -v "test-varlink-idl" >>/failed)
+(! journalctl -q -o short-monotonic --grep "didn't pass validation" | grep -v "test-varlink-idl" | tee --append /failed)
```

Comment:
Potentially, this makes matching lines stored to journal, and they are shown and matches again again again ?

Reply:
That is true, will drop this commit